src/main/java/eu/execom/pomodoroTeam/services/UserDaoImpl.java

```diff
@@ -26,5 +21,5 @@
         List<UserEntity> result = new ArrayList<UserEntity>();
         result = query.getResultList();
         return result;
-    }
+    }*/
```

Comment:
This class should be removed, but please have in mind that no commented out code should be pushed to the code base.